app/models/manageiq/providers/kubernetes/container_manager.rb

```diff
@@ -41,6 +41,10 @@ def self.description
     @description ||= "Kubernetes".freeze
   end
 
+  def self.fonticon
+    @fonticon ||= "pficon-kubernetes".freeze
```

Comment:
Can it just be derived on the UI side without needing a model change?  seems like the UI can just do `"pficon-#{@ems.ems_type}"` or something similar to how we do it for other images.